ckan/cli/generate.py

```diff
@@ -77,3 +80,32 @@ def extension(output_dir):
 
     cookiecutter(template_loc, no_input=True, extra_context=context,
                  output_dir=output_dir)
+
+
+@generate.command(name=u'config',
+                  short_help=u'Path to the generated configuration ini file.')
```

Comment:
This is the help to the config command, not its param:
```
$ ckan generate --help
Usage: ckan generate [OPTIONS] COMMAND [ARGS]...

Options:
  --help  Show this message and exit.

Commands:
  config     Path to the generated configuration ini file.
  extension  Create empty extension.
```
How about
```suggestion
                  short_help=u'Create a ckan.ini file.')
```

Reply:
@davidread Ah, I missed that. Okay, will do. Thanks!

Reply:
@davidread Okay. I took care of those in my latest commit.